lib/bots/versionbot.ts

```diff
@@ -980,18 +998,9 @@ export class VersionBot extends ProcBot {
 				});
 			}
 		}).then(() => {
-			// Get the labels for the PR.
-			return this.dispatchToEmitter(this.githubEmitterName, {
-				data: {
-					number: pr.number,
-					owner,
-					repo: name
-				},
-				method: this.githubApi.issues.getIssueLabels
-			});
-		}).then((labels: GithubApiTypes.IssueLabel[]) => {
-			// If we don't have a relevant label for merging, we don't proceed.
-			if (_.some(labels, (label) => label.name === MergeLabel)) {
+			// If we don't have a relevant label for merging (or we do but the PR is marked as a WIP),
+			// we don't proceed.
+			if (VersionBot.hasLabel(labels, MergeLabel) && !VersionBot.hasLabel(labels, WIPLabel)) {
```

Comment:
The code and comment above depend on the label name, which can be looked up. I'd rather change one instance of `flow/in-progress` rather than that and comments which could be missed in future (as already proved elsewhere). If you look at the Commit/PR guidelines, it's explicitly mentioned there in the workflow for engineers.